pkgs/development/tools/misc/linuxkit/default.nix

```diff
@@ -1,29 +1,29 @@
 { lib, buildGoPackage, fetchFromGitHub }:
 
 buildGoPackage rec {
-  name = "linuxkit-${version}";
-  version = "2017-07-08";
-  rev = "8ca19a84d5281b1b15c7a48c21e5786943b47f1c";
+  pname = "linuxkit";
+  version = "0.7";
 
   goPackagePath = "github.com/linuxkit/linuxkit";
 
   src = fetchFromGitHub {
     owner = "linuxkit";
     repo = "linuxkit";
-    inherit rev;
+    rev = "4f4f7feae125e1b2796806fff708fd3ac9c20657";
     sha256 = "150y7hnjhi81iik7np27y5466ldaackq72mpi7vmybbl7vr1wgw4";
   };
 
   subPackages = [ "src/cmd/linuxkit" ];
 
   preBuild = ''
-    buildFlagsArray+=("-ldflags" "-X main.GitCommit=${builtins.substring 0 7 rev} -X main.Version=0.0.0")
+    buildFlagsArray+=("-ldflags" "-X main.GitCommit=${src.rev} -X main.Version=v${version}")
```

Comment:
For the sake of automatic updates via @r-ryantm I would just keep the GitCommit empty here so we don't need to update the revision ourself.

Reply:
Makes sense.  It ends up looking like this, which doesn't seem so bad.
```
% result-bin/bin/linuxkit version
linuxkit version v0.7
commit: unknown
```